### ISSUE TITLE
Latest master branch module registration problem

### ISSUE THREAD

**Issue by ChrisSun1212** [simplcommerce/SimplCommerce#270]
I am using Visual Studio 2017 preview and when executing "update-database" in NuGet package manager, I got the following error

An error occurred during the activation of a particular registration. See the inner exception for details. Registration: Activator = SimplDbContext (DelegateActivator), Services = [SimplCommerce.Module.Core.Data.SimplDbContext], Lifetime = Autofac.Core.Lifetime.CurrentScopeLifetime, Sharing = Shared, Ownership = OwnedByLifetimeScope ---> Could not load file or assembly 'System.Diagnostics.DiagnosticSource, Version=4.0.2.1, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'. The system cannot find the file specified. (See inner exception for details.)

Thanks,

Chris

**Comment by thiennn** [comment]
Please make sure that you have VS 15.3 or newer version, and dotnet core 2.0 sdk installed. Then checkout the https://github.com/simplcommerce/SimplCommerce/wiki/Development-tips

**Comment by ChrisSun1212** [comment]
Hi, I followed your instructions closely and had VS15.3 and dotnet core 2.0 sdk installed. However, the problem persisted. Interestingly your config20 branch worked without any problem. Btw, there is one more problem. When I tried to start the project on VS by using the "SimplCommerce.WebHost" option directly, the project failed to start with an Invalid Uri exception. Is there any suggestion on how to fix this?

Thank you very much,

Chris
